src/Pipfile.lock

```diff
@@ -1685,4 +1685,4 @@
             "version": "==3.0.0"
         }
     }
-}
+} 
```

Comment:
```suggestion
}
```

Reply:
+1 to removing the accidental code change